src/fetchers/tarball-fetcher.js

```diff
@@ -208,11 +216,7 @@ export default class TarballFetcher extends BaseFetcher {
       return this.fetchFromLocal(this.reference);
     }
 
-    if (await this.getLocalAvailabilityStatus()) {
-      return this.fetchFromLocal();
-    } else {
-      return this.fetchFromExternal();
-    }
+    return this.fetchFromLocal().catch(err => this.fetchFromExternal());
```

Comment:
My only concern around this now is with offline mode, I'll double check that this change doesn't affect that.